com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -0,0 +1,66 @@
+---
+id: com.github.jeromerobert.pdfarranger
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: pdfarranger
+# These renames will be obsolete with 
+# https://github.com/jeromerobert/pdfarranger/issues/158
+# Currently known bug: Missing icon in about dialog (Will be fixed by #158.)
+rename-icon: pdfarranger
+rename-desktop-file: pdfarranger.desktop
+rename-appdata-file: pdfarranger.appdata.xml
+finish-args:
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --filesystem=home
+modules:
+- shared-modules/intltool/intltool-0.51.json
+- shared-modules/openjpeg/openjpeg.json # dependency of poppler
+- name: poppler
```

Comment:
You probably want a `cleanup` section for poppler too.

Reply:
For pip generator see my [other comment](https://github.com/flathub/flathub/pull/1427#discussion_r400793681) 
I'll look into cleaning up after poppler...